.gitignore

```diff
@@ -1,10 +1,10 @@
 # Build products
-/eos-label-location
-/eos-metrics-instrumentation
-/eos-network-id
+/src/eos-metrics-instrumentation
```

Comment:
I still think you should use [`git.mk`](https://github.com/behdad/git.mk) in a follow-up PR or commit. It’s really really awesome.